admin_guide/managing_projects.adoc

```diff
@@ -94,3 +101,55 @@ or:
 To request a new project, fill out the project request form located at
 https://internal.example.com/openshift-project-request.
 ----
+
+== Node Selectors
```

Comment:
I think "using Node Selectors" is fine here. This is kinda the first point for node selectors (which is why I kept it as "Node Selectors" in the first place), and any other times they might be mentioned in anchors they would have a "Using Node Selectors for X" in the title, or something similar.